src/utility.js

```diff
@@ -71,6 +71,19 @@ function isFunction(f) {
   return isType(f, 'function');
 }
 
+/* isNativeFunction - a convenience function for checking if a value is a native JS function
+ *
+ * @param f - any value
+ * @returns true if f is a native JS function, otherwise false
+ */
+function isNativeFunction(f) {
+  if (!isFunction(f)) {
+    return false;
+  }
+  var functionString = Function.prototype.toString.call(f);
+  return /\{\s+\[native code\]/.test(functionString);
+}
```

Comment:
I tested it on IE9 and it works, however im struggling to find any coverage reports on the usage of `[native code]`. I've been trying to test it in IE6 manually but i cant open firebug using browser stack. Do you have an cross browse tests i can write ?